features/object_mapper/creating/README.md

```diff
@@ -43,9 +43,9 @@ The class must provide a default constructor, and all fields (except the
 ones annotated `@Transient`) must have corresponding setters and
 getters.
 
-[table]:http://docs.datastax.com/en/drivers/java/2.2/com/datastax/driver/mapping/annotations/Table.html
-[case-sensitive]:http://docs.datastax.com/en/cql/3.2/cql/cql_reference/ucase-lcase_r.html
-[consistency level]:http://docs.datastax.com/en/drivers/java/2.2/com/datastax/driver/core/ConsistencyLevel.html
+[table]:http://docs.datastax.com/en/drivers/java/3.0/com/datastax/driver/mapping/annotations/Table.html
+[case-sensitive]:http://docs.datastax.com/en/cql/3.1/cql/cql_reference/ucase-lcase_r.html
```

Comment:
You'll probably encounter this in the rebase, but this has been updated to 3.3 from the upstream branches (2.0/2.1) and that seems to work good.